### DISCUSSION TITLE
Default filters

### DISCUSSION THREAD

**Discussion by MartinMatko** [jgraichen/redmine_dashboard#152] [Ideas]
Hi developers of great redmine dashboard!
I'd like to ask if you thought about possibility to configure default filters (or put them in url)
so everytime I start dashboard, I don't have to set up filters again and again.
It would be great for me (and other scrum masters and developers as well of course)

Best regards,
Martin

**Comment by jgraichen** [comment]
Some sort of default filter can be configured in [`config/default.yml`](https://github.com/jgraichen/redmine_dashboard/blob/main/config/default.yml). These will be used when a new user, or a user with a new session, opens any dashboard.

This can help with a global default for dashboard.
